conans/client/file_copier.py

```diff
@@ -138,7 +138,22 @@ def _filter_files(src, pattern, links, excludes, ignore_case, excluded_folders):
                     subfolders[:] = []
                     files = []
                     break
+
             for f in files:
+                abs_path = os.path.join(root, f)
+
+                if os.path.islink(abs_path):
+                    linkto = os.path.join(os.path.dirname(abs_path), os.readlink(abs_path))
+                    linkto = os.path.normpath(linkto)
+                    if not os.path.exists(linkto):
+                        self._output.warn("File '{}' is pointing to '{}' that doesn't exists. It"
```

Comment:
I think it's still valid, e.g. I may have a symlink pointing to the `/dev/null`, which will be find after unpack, right?

Reply:
`os.path.exists("/dev/null")` returns True, but it will be out of the source folder and discarded in the next check. Maybe this is an exception and we want to keep a symlink to the special path `/dev/null`?